src/fshtypes/rules/InsertRule.ts

```diff
@@ -17,7 +17,15 @@ export class InsertRule extends Rule {
   private fshifyParameters(): string {
     return this.params
       .map(originalParam => {
-        return originalParam.replace(/\\/g, '\\\\').replace(/,/g, '\\,').replace(/\)/g, '\\)');
+        if (/(^\s+)|(\s+$)|[,\)]/g.test(originalParam)) {
+          return `[[${originalParam
+            .replace(/\\/g, '\\\\')
+            .replace(/(\]\]\s*),/g, '$1\\,')
+            .replace(/(\]\]\s*)\)/g, '$1\\)')}]]`;
```

Comment:
I think you can swap the last two replaces with a single replace by making another capture group:
```ts
.replace(/(\]\]\s*)([,)])/g, '$1\\$2')
```